src/hacbs/components/Commits/__mocks__/pipeline-with-commits.ts

```diff
@@ -0,0 +1,725 @@
+import { PipelineRunKind } from '../../../types';
```

Comment:
`__mocks__` directory is used for [mocking a component or a module in jest](https://jestjs.io/docs/manual-mocks#mocking-user-modules). Let us use `__data__` to place test data.

Reply:
updated